features/support/time_machine_helpers.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module TimeMachineHelpers
+  def time_machine(till)
```

Comment:
I think we should add some comments to this helper Module.
It has one public method called `time_machine` and travels in time, runs jobs defined in `ThreeScale::Jobs` and travel back. Maybe the jobs we're running should be passed as arguments to `time_travel` or, if it's here for the purpose of running the jobs, we should rename the method to something like `run_jobs_till(till)`.

Reply:
Yeah, this module was already there and I renamed it because it used to reference `Timecop` in the name. It's purpose is not to set a new time but to simulate the pass of time with all event in the middle, that's why it runs the jobs. For instance: if you remove a user, the user is not really removed after a month later, so in a test you can do something like:

1. Remove the user
2. Time_machine 1 month from now
3. Assert the user is effectively removed

It could be called `run_jobs_till` but I find `time_travel` better because it's more sci-fi 😎. don't have a strong opinion, though.

Reply:
I get the concept now that you explained it but, by looking just at the code, I see jobs being executed while time passes 😅.
If we can't find a better name for the method (better = easier to understand what it does), a comment to this module should be good enough 👍

Reply:
I'll add a comment then